nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -391,6 +390,11 @@ public boolean hasFeature(Feature feature) {
         }
     }
 
+    @Override
+    public DocumentRef getTopLevelCommentAncestor(CoreSession session, DocumentRef commentIdRef) {
+        return getAncestorRef(session, session.getDocument(commentIdRef));
```

Comment:
Do you need to keep `getAncestorRef` ?

Reply:
as it is a `protected` method and to keep the backward compatibility, i propose to keep the method but to deprecate it 
wdyt

Reply:
I think there's no need to deprecate `protected` methods in final implementation, it may worth it for the abstract class but this is not the case here. You can clean such things there as these services has become more and more verbose.

Reply:
i see, it is true that we are in a case of implementation and i think there is no many cases where customer use directly or inherent the `PropertyCommentManager`